code/modules/clothing/spacesuits/plasmamen.dm

```diff
@@ -12,28 +12,84 @@
 	max_heat_protection_temperature = SPACE_SUIT_MAX_TEMP_PROTECT
 	species_restricted = list("Plasmaman")
 	flags = STOPSPRESSUREDMAGE
+	autoextinguish = 1
 
 	icon_state = "plasmaman_suit"
 	item_state = "plasmaman_suit"
 
+	actions_types = list(/datum/action/item_action/eject_depleted_cartridges)
```

Comment:
In that case the spare cartridges would be spare equipment the Plasmamen spawned in with correct? Just got back